app/models/intervention_output.rb

```diff
@@ -93,6 +93,15 @@ class InterventionOutput < InterventionProductParameter
     end
   end
 
+  def choose_default_name(output)
+    if output.name.eql?("blank")
+      land_parcel_name = self.group.targets.first.product.name rescue ""
+      culture_name = self.variant.name rescue ""
+      output.name = "#{land_parcel_name} #{culture_name} #{self.group.intervention.created_at.strftime('%Y')} #{self.group.intervention.id}"
```

Comment:
This should be the intervention.number and not id as the user won't recognize those.